### ISSUE TITLE
Improve relative dates

### ISSUE THREAD

**Issue by aiscarvalho** [beabee-communityrm/beabee-frontend#300]
First drafts for a component that tries to condense a variety of options for date inputs in the advanced search:

![image](https://user-images.githubusercontent.com/122730/199749769-2e113aa4-a8e6-4983-9ce3-73abe2c757f8.png)

![image](https://user-images.githubusercontent.com/122730/199750030-59124005-00a1-46bf-9e2e-2f7a4de5eb2d.png)

[Penpot document](https://design.penpot.app/#/view/448268f0-2f80-11ed-9e6a-c67f7655c7b3?page-id=78312ce8-0ae7-80c9-8001-8abd1389ba18&section=interactions&index=0&share-id=69cd8e21-398c-8036-8001-8abdd2a04a46)

**Comment by wpf500** [comment]
@aiscarvalho how would it work to set relative dates in the future? i.e. -3 days vs +3 days

**Comment by aiscarvalho** [comment]
@wpf500 we edited the [spec](https://design.penpot.app/#/view/448268f0-2f80-11ed-9e6a-c67f7655c7b3?page-id=78312ce8-0ae7-80c9-8001-8abd1389ba18&section=interactions&index=1&share-id=69cd8e21-398c-8036-8001-8abdd2a04a46) to contemplate future dates:

Adding Past/Future as options in the dropdown:
![image](https://user-images.githubusercontent.com/122730/201147277-2626f395-2d9c-46fe-be23-55137026ed7b.png)

An example query setting an interval between a past date and a future date:
![image](https://user-images.githubusercontent.com/122730/201147503-382e374c-dc24-4ce2-b05a-53ef572ebe08.png)

Another example query with a future date:
![image](https://user-images.githubusercontent.com/122730/201149900-6d80d818-3451-4fc2-a60a-a527e6a1af8b.png)

We think this would be the simplest way to add future to the relative date selector we proposed. The only thing we're not happy with is the "ago/ahead" but couldn't come up with a better wording for it.
Maybe we could brainstorm together about the best way to phrase this?
